itests/kit/evm.go

```diff
@@ -310,3 +324,40 @@ func removeLeadingZeros(data []byte) []byte {
 	}
 	return data[firstNonZeroIndex:]
 }
+
+func SetupFEVMTest(t *testing.T) (context.Context, context.CancelFunc, *TestFullNode) {
+	//make all logs extra quiet for fevm tests
+	lvl, err := logging.LevelFromString("error")
+	if err != nil {
+		panic(err)
+	}
+	logging.SetAllLoggers(lvl)
+
+	blockTime := 100 * time.Millisecond
+	client, _, ens := EnsembleMinimal(t, MockProofs(), ThroughRPC())
+	ens.InterconnectAll().BeginMining(blockTime)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+
+	// require that the initial balance is 100 million FIL in setup
+	// this way other tests can count on this initial wallet balance
+	fromAddr := client.DefaultKey.Address
+	bal, err := client.WalletBalance(ctx, fromAddr)
+	require.NoError(t, err)
+	originalBalance := types.FromFil(100 * uint64(1e6)) // 100 million FIL
```

Comment:
In golang, you can declare ints with underscores in them, so usually we would declare this as `100_000_000', which is readable and also concise.